src/Physics/Evaluate_friction_law.f90

```diff
@@ -1300,31 +1302,33 @@ SUBROUTINE rate_and_state_nuc103(TractionGP_XY,TractionGP_XZ,        & ! OUT: tr
          !                                      mu_ss = mu_w + [mu_lv - mu_w] / [ 1 + (V/Vw)^8 ] ^ (1/8) ]
          !                                      mu_lv = mu_0 - (b-a) ln (V/V0)
          !
-         LocP   = NorStressGP(:,iTimeGP)
+         LocP   = 0.8*NorStressGP(iBndGP,iTimeGP)
```

Comment:
Where does this factor come from?

Reply:
The title may be misleading. 
It is just for Sebastian and our group to test whether it fix his bugs. 
It doesn't need to be merged. 

The factor can be moved.